src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const mediumTerm = 3;
+  const mediumTermDiscount = 20;
+
+  let totalRent = 0;
```

Comment:
Consider using `const` instead of `let` for `totalRent` since its value is not being reassigned after its initial calculation. This helps to prevent unintentional reassignments and maintain the immutability of the variable.